x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/components/steps/components/hooks.tsx

```diff
@@ -34,7 +34,9 @@ export function useOutputs(
     if (!outputsData || !canUseOutputPerIntegration) {
       return [];
     }
-    return outputsData.items.filter((output) => allowedOutputTypes.includes(output.type));
+    return outputsData.items.filter(
+      (output) => allowedOutputTypes.includes(output.type) && !output.is_preconfigured
```

Comment:
we should not filter on `is_preconfigured` as it's possible to define preconfigured output for users, for example it's what we use.